src/config.ts

```diff
@@ -0,0 +1,31 @@
+import {CONTRACT_VERSION} from "./contract/constants";
+import {ContractVersionValue} from "./contract/types";
+import {encodeString} from "./util/util";
+
+class TinymanJSSDKConfig {
+  /** Identifier name of the SDK user */
+  clientName: string;
+
+  constructor() {
+    this.clientName = "tinyman-js-sdk";
+  }
+
+  getClientName() {
+    return this.clientName;
+  }
+
+  setClientName(name: string) {
+    this.clientName = name;
+  }
+
+  /**
+   * @returns {Uint8Array} - encoded note includings version with client name
+   */
+  getAppCallTxnNoteWithClientName(contractVersion: ContractVersionValue): Uint8Array {
+    const versionMarker = contractVersion === CONTRACT_VERSION.V1_1 ? "v1" : "v2";
```

Comment:
I think we can fallback to contractVersion so that when other versions become available it would just work (hopefully). 
```suggestion
    const versionMarker = contractVersion === CONTRACT_VERSION.V1_1 ? "v1" : contractVersion;
```